src/main/java/seedu/address/MainApp.java

```diff
@@ -118,12 +121,12 @@ protected Config initConfig(Path configFilePath) {
         try {
             Optional<Config> configOptional = ConfigUtil.readConfig(configFilePathUsed);
             if (!configOptional.isPresent()) {
-                logger.info("Creating new config file.");
+                logger.info("Creating new config file " + configFilePathUsed);
             }
             initializedConfig = configOptional.orElse(new Config());
         } catch (DataConversionException e) {
-            logger.warning("Config file at " + configFilePathUsed + " is not in the correct format. "
-                    + "Using default config properties");
+            logger.warning("Config file at " + configFilePathUsed
+                    + " is not in the correct format. Using default config properties.");
```

Comment:
Previous line wrapping position is better?

Reply:
For consistency, wrap when you are close to the max length. Don't wrap in arbitrary places for the sake of readability unless there is a strong justification. 
Reason: While readability is good, line wrapping differently to improve readability would cause us to make case-by-case decisions every time we do line wrapping. It's not worth the effort.